operator/pom.xml

```diff
@@ -89,6 +89,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-openshift-client</artifactId>
+        </dependency>
```

Comment:
OpenShift client was removed in https://github.com/quarkiverse/quarkus-operator-sdk/pull/1053